js/packages/web/src/utils/analytics.ts

```diff
@@ -0,0 +1,115 @@
+// a test GA4 Id to see if it works in the staging test app
```

Comment:
I think this would all be great to have bundled in a single analytics provider vs util functions.